packages/snaps-utils/src/json-rpc.ts

```diff
@@ -12,11 +12,22 @@ import {
 import type { Infer } from 'superstruct';
 import { array, boolean, object, optional, refine, string } from 'superstruct';
 
+const AllowedOriginsStruct = array(
+  refine(string(), 'Allowed origin', (value) => {
+    const wildcards = value.split('*').length - 1;
+    if (wildcards > 2) {
+      return 'No more than two wildcards (*) are allowed in "allowedOrigins".';
```

Comment:
```suggestion
      return 'No more than two wildcards ("*") are allowed in an origin specifier.';
```
Or something similar? Otherwise it may imply that the combined wildcards in all allowed origins may not be >2.

Reply:
Yeah, that's a good point!